data/migrations/20200204164931_cars-schema.js

```diff
@@ -0,0 +1,12 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('cars', tbl =>{
+      tbl.increments();
+      tbl.text('Make', 128).unique().notNullable();
+      tbl.text('Model', 128).unique().notNullable();
+      tbl.decimal('year');   
+  })
+};
```

Comment:
make sure that you are tracking the following

VIN
Make
Model
Mileage

The following is not always known at the time of entry, but make sure you find a way to make it possible to not be required 😉 

Transmission type
Status of Title